### DISCUSSION TITLE
Create and import runs automatically via script?

### DISCUSSION THREAD

**Discussion by kiraschweickhardt** [rubenarslan/formr.org#538] [Q&A]
For my job I need to create about two hundred Runs and import the corresponding 200 json-files (which are created in an R-Script). Since it takes a long time to type in every Run-name individually and make sure to upload the right json-file for each of them, I was wondering if there might be a way to automatically upload several runs - maybe from the same R-script? I am looking for something similar to the function formr_upload_items, but for uploading runs instead. Thank you in advance!

**Comment by rubenarslan** [comment]
This does not yet exist, but it could be scripted in R like you suggest. Happy to help, for a fee.
